### ISSUE TITLE
Show Location Data of  Video like photos.

### ISSUE THREAD

**Issue by nikhilbadyal** [SimpleMobileTools/Simple-Gallery#2256]
As the title says 
SG shows location data present in photos when we press the ℹ️ in the photo. But video also have location within them. How about a similar feature for them.

**Comment by tibbi** [comment]
What app and OS version are you using? Do other apps show the coordinates well?

**Comment by nikhilbadyal** [comment]
I haven't found any gallery with this feature so kind of exclusive. 

And as far ik GPS/Location are embedded in QuickTime/XMP unlike EXIF in photos.

So if this is within limits. Please have a look.

**Comment by tibbi** [comment]
if you send the file to yourself via email and download it, does the new file still show the location?

**Comment by nikhilbadyal** [comment]
Yes, I emailed myself a video file from other email account and as expected Location coordinates were there.

I used exiftool to check the metadata. 
> ExifTool Version Number         : 12.33
..
Many more details 
..
GPS Latitude                    : 32 deg 41' 44.88" N
GPS Longitude                   : 74 deg 51' 48.60" E
Rotation                        : 90
GPS Position                    : 32 deg 41' 44.88" N, 74 deg 51' 48.60" E

**Comment by nikhilbadyal** [comment]
I realized `google photos` do show location in videos.
